### PR TITLE
Members list

### PR DESCRIPTION
**Pull request by Shaliapyn**

### CODE REVIEW

**Review by d-aliona — COMMENTED** [review]
import ManagerManagment from "./pages/ManagerManagment/ManagerManagment";
В App.js цей імпорт задовгий. Ти ж створив index.js.

File: src/App.js [hunk 1]
```diff
@@ -3,13 +3,13 @@ import "bootstrap/dist/css/bootstrap.css";
...
 
+import Layout from "./ui/Layout"
 import Start from "./pages/Start";
 import Profile from "./pages/Profile";
 import MemberList from "./pages/MemberList";
 import ManagerList from "./pages/ManagerList";
 import EventList from "./pages/EventList/";
-import Layout from "./ui/Layout";
-import Event from "./pages/Event";
+import ManagerManagment from "./pages/ManagerManagment/ManagerManagment";
```
Comment (nazarii26) [inline src/App.js]: you should make shorter link

File: src/App.js [hunk 2]
```diff
@@ -20,8 +20,9 @@ function App() {
           <Route path="/profile" element={<Profile />} />
           <Route path="/memberList" element={<MemberList />} />
           <Route path="/managerList" element={<ManagerList />} />
+          <Route path="/managerManagment" element={<ManagerManagment />} />
           <Route path="/eventList" element={<EventList />} />
-          <Route path="/event" element={<Event />} />
+          <Route path="/event" element={<ManagerList />} />
```
Comment (nazarii26) [inline src/App.js:25]: <Route path="/event" element={<ManagerList />} />
  
  maybe we should change path="/event" to path="/managerList" ?

File: src/pages/ManagerManagment/ManagerManagment.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+              <th scope="col">Last Name</th>
+              <th scope="col">Email</th>
+              <th scope="col">Managment</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr className={styles.tableRow}>
+              <th>Avatar</th>
+              <td>Gustavus</td>
+              <td>Adolphus</td>
```
Comment (nazarii26) [inline src/pages/ManagerManagment/ManagerManagment.js:30]: funny :)

**Review by nazarii26 — COMMENTED** [review]
good job!

but you should make shorter links via index.js

**Pull request merged**